modules/ROOT/pages/external-org-access.adoc

```diff
@@ -0,0 +1,41 @@
+= External Organization Access
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Organizations that have their own, separate domains in Anypoint Platform can share resources with one another using external access lists of trusted organizations.
```

Comment:
I don't know how aware customers are of domains. Do we introduce the concept to them anywhere?

Reply:
It's introduced here https://docs.mulesoft.com/access-management/organization#manage-master-organization-settings
but it wouldn't hurt to reacquaint them with the term.